### ISSUE TITLE
Vega event selector syntax for the event listeners

### ISSUE THREAD

**Issue by jhoffswell** [vega/vega#352]
I was trying to use ```ved.view.on(...)``` to add event listeners and was wondering if it could support targeting like ```@cell:mousedown``` in addition to regular events like ```mousemove```.

**Comment by jheer** [comment]
I believe these listen directly to the standard DOM-style events provided by the scenegraph handlers. @arvind, how hard would it be to add event stream selectors here?

**Comment by arvind** [comment]
Yup, `view.on` functions like an `addEventListener` call and accepts standard DOM events. Extending it to accept event stream selectors shouldn't be too difficult, but will require some tweaks to `parse/streams.js.` In particular, we will need to do some light refactoring to allow it to be called in a "one-off" fashion for listeners (may be as simple as exposing the `mergedStream` function). I also need to think through exactly when additional signals need to be added and when calling the event handler directly would be fine. 

I've added this feature request under a v2.3 milestone.
